BeagleBone/AI/pru/Makefile

```diff
@@ -0,0 +1,92 @@
+# 
+# Copyright (c) 2016 Zubeen Tolani <ZeekHuge - zeekhuge@gmail.com>
+# Copyright (c) 2017 Texas Instruments - Jason Kridner <jdk@ti.com>
+# 
+
+# TARGET must be defined as the file to be compiled without the .c.
+# PRUN must be defined as the PRU number (0 or 1) to compile for.
+
+# PRU_CGT environment variable points to the TI PRU compiler directory.
+# PRU_SUPPORT points to pru-software-support-package.
+# GEN_DIR points to where to put the generated files.
+
+$(warning TARGET=$(TARGET), PRUN=$(PRUN), MODEL=$(MODEL))
+
+PRU_CGT:=/usr/share/ti/cgt-pru
+PRU_SUPPORT:=/usr/lib/ti/pru-software-support-package
+GEN_DIR:=/tmp/pru$(PRUN)-gen
+COMMON:=/var/lib/cloud9/BeagleBone/AI/pru
+
+LINKER_COMMAND_FILE=AM335x_PRU.cmd
+LIBS=--library=$(PRU_SUPPORT)/lib/rpmsg_lib.lib
+INCLUDE=--include_path=$(PRU_SUPPORT)/include --include_path=$(PRU_SUPPORT)/include/am335x
+
+STACK_SIZE=0x100
+HEAP_SIZE=0x100
+
+CFLAGS=-v3 -O2 --printf_support=minimal --display_error_number --endian=little --hardware_mac=on --obj_directory=$(GEN_DIR) --pp_directory=$(GEN_DIR) --asm_directory=$(GEN_DIR) -ppd -ppa --asm_listing --c_src_interlist -DAI=$(AI) # --absolute_listing
+
+LFLAGS=--reread_libs --warn_sections --stack_size=$(STACK_SIZE) --heap_size=$(HEAP_SIZE) -m $(GEN_DIR)/$(TARGET).map
+
+# Check which model
+PRU_DIR="Invalid PRUN"
+ifeq ($(MODEL), AI)
```

Comment:
Can we avoid coming up with new strings to compare and just use the model name it already depends on?